RELEASE.md

```diff
@@ -10,32 +10,36 @@
 
 Start a cluster using `cluster-bot` application.
 
-To be able to test update it is needed to created some user before. Login as `kubeadmin`. At the middle of the dashboard click `Update OAuth configuration`. Then `Identity providers` -> `Add` -> `HTPassword` and upload a htpassword file (can be created with HTPassword utility). Login using HTPassword, logout and login again as `kubeadmin`. Go to `kube:admin` menu -> `Copy Login Command` -> `Display Token` and launch to showing command in the terminal. Now it is possible to test update:
+To be able to test update it is needed to created some user before. Login as `kubeadmin`. Click `Update the cluster OAuth configuration` at the middle of the dashboard, then `Identity providers` -> `Add` -> `HTPassword` and upload a htpassword file (can be created with HTPassword utility). Logout and login using HTPassword, then logout and login as `kubeadmin`. Go to `kube:admin` -> `Copy Login Command` -> `Display Token` and launch showing command in the terminal. Now it is possible to test update:
 
 ```bash
-olm/testUpdates.sh openshift stable
+olm/testUpdate.sh openshift stable
 ```
 
-Open Eclipse Che dashboard in the anonymous to validate that the correct version is installed and workspace can be created:
+Open Eclipse Che dashboard in an anonymous tab:
 
 ```bash
-echo http://$(kubectl get ingress -n che | grep ^che | awk -F ' ' '{ print $2 }')
+echo http://$(oc get route -n eclipse-che-preview-test | grep ^che | awk -F ' ' '{ print $2 }')
```

Comment:
The even simpler solution is to retrieve the URL from the status of the `CheCluster` CR when it is in the `Available` state.

Reply:
Agree, I will update the script and use it in the next release.